test/e2e/serial/test_suite_serial_test.go

```diff
@@ -59,8 +67,9 @@ var (
 // and indisturbed on the cluster. No concurrency at all is possible,
 // each test "owns" the cluster - but again, must leave no leftovers.
 
-// do not use this fixture outside this *file*
+// do not use these outside this *file*
 var __fxt *e2efixture.Fixture
+var __nrtList nrtv1alpha1.NodeResourceTopologyList
```

Comment:
It's a none-issue but I must ask, what's the double underscore purpose here?

Reply:
it's a way to convey this identifier is private-private and should never be used not even outside the same package